src/decoders.js

```diff
@@ -1,4 +1,4 @@
-let outlook_regex = /https:\/\/\w+\.safelinks\.protection\.outlook\.com\/\?url=(\S+?)((?:\&amp;data.*)+$|$)/i;
+let outlook_regex = /https:\/\/\w+\.safelinks\.protection\.outlook\.com\/\?url=([^&]*)(&[-+a-zA-Z0-9%&/=.;]*)?/i;
```

Comment:
Shouldn't the `+` and `.` be escaped above?
```suggestion
let outlook_regex = /https:\/\/\w+\.safelinks\.protection\.outlook\.com\/\?url=([^&]*)(&[\-+a-zA-Z0-9%&/=\.;]*)?/i;
```

Reply:
Don't ask me when exactly they need to be escaped. This seems to work fine. But there is more to improve, e.g. excluding trailing `.`.